glusterd2/commands/volumes/volume-profile-utils.go

```diff
@@ -0,0 +1,100 @@
+package volumecommands
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/volume"
+)
+
+// Fops
+var fops = []string{
+	"NULL",
+	"STAT",
+	"READLINK",
+	"MKNOD",
+	"MKDIR",
+	"UNLINK",
+	"RMDIR",
+	"SYMLINK",
+	"RENAME",
+	"LINK",
+	"TRUNCATE",
+	"OPEN",
+	"READ",
+	"WRITE",
+	"STATFS",
+	"FLUSH",
+	"FSYNC", /* 16 */
+	"SETXATTR",
+	"GETXATTR",
+	"REMOVEXATTR",
+	"OPENDIR",
+	"FSYNCDIR",
+	"ACCESS",
+	"CREATE",
+	"FTRUNCATE",
+	"FSTAT", /* 25 */
+	"LK",
+	"LOOKUP",
+	"READDIR",
+	"INODELK",
+	"FINODELK",
+	"ENTRYLK",
+	"FENTRYLK",
+	"XATTROP",
+	"FXATTROP",
+	"FGETXATTR",
+	"FSETXATTR",
+	"RCHECKSUM",
+	"SETATTR",
+	"FSETATTR",
+	"READDIRP",
+	"FORGET",
+	"RELEASE",
+	"RELEASEDIR",
+	"GETSPEC",
+	"FREMOVEXATTR",
+	"FALLOCATE",
+	"DISCARD",
+	"ZEROFILL",
+	"IPC",
+	"SEEK",
+	"LEASE",
+	"COMPOUND",
+	"GETACTIVELK",
+	"SETACTIVELK",
+	"PUT",
+	"ICREATE",
+	"NAMELINK",
+	"MAXVALUE",
+}
+
+var profileSessionKeys = [...]string{"io-stats.count-fop-hits", "io-stats.latency-measurement"}
+
+// BrickProfileInfo holds profile info of each brick
+type BrickProfileInfo struct {
+	BrickName       string   `json:"brick-name"`
+	CumulativeStats StatType `json:"cumulative-stats,omitempty"`
+	IntervalStats   StatType `json:"interval-stats,omitempty"`
+}
+
+// StatType contains profile info of cumulative/interval stats of a brick
+type StatType struct {
+	Duration             string                       `json:"duration"`
+	DataRead             string                       `json:"data-read"`
+	DataWrite            string                       `json:"data-write"`
+	Interval             string                       `json:"interval"`
+	PercentageAvgLatency float64                      `json:"total-%-avg-latency"`
```

Comment:
can we rename this JSON name to something else, IMO using ```%```  and more hyphens  in JSON name is not good for readability